api/approx_count_distinct.md

```diff
@@ -0,0 +1,59 @@
+---
+api_name: approx_count_distinct()
+excerpt: Aggregate data into a hyperloglog for approximate counting without having to specify the number of buckets.
+license: community
+toolkit: true
+topic: hyperfunctions
+tags: [hyperfunctions, approximate count distinct, distinct count, hyperloglog]
+api_category: hyperfunction
+toolkit-experimental: false
+hyperfunction_toolkit: true
+hyperfunction_family: 'approximate count distinct'
+hyperfunction_subfamily: hyperloglog
+hyperfunction_type: aggregate
+---
+
+# approx_count_distinct()  <tag type="toolkit">Toolkit</tag>
+The `approx_count_distinct` function constructs and returns a hyperloglog with at least
+32,768 buckets over the given values.
+
+For more information about approximate count distinct functions, see the
+[hyperfunctions documentation][hyperfunctions-approx-count-distincts].
+
+## Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|value|AnyElement| Column to count distinct elements. The type must have an extended, 64-bit, hash function.|
+
+Since hyperloglog is a probabilistic algorithm, it works best on datasets that have many distinct values: at least tens of thousands. 
```

Comment:
We probably should remove this as well, since we should get exact match for anything less than 32k values.